### ISSUE TITLE
How to enable for remote browser?

### ISSUE THREAD

**Issue by andrewhavens** [jfirebaugh/capybara-firebug#1]
I am using selenium with a remote browser (cucumber is executed remotely, and the browser triggered is on my desktop). How can I get this to work when it is triggering a remote browser? Here's what I've got in my env.rb:

```
# Capybara configuration (using Selenium)
require 'capybara'
require 'capybara/dsl'
require 'capybara/cucumber'
require 'capybara/firebug'

Capybara.default_driver = :selenium
Capybara.javascript_driver = :selenium
Capybara.run_server = false
#Capybara.app_host = "https://google.com" # this is replaced in my Before hook
Capybara.register_driver :selenium do |app|
  Capybara::Driver::Selenium.new(app,
    :browser => :remote,
    :url => "http://my.ip.add.ress:4444/wd/hub",
    :desired_capabilities => :firefox)
end

World(Capybara)
```

**Comment by jfirebaugh** [comment]
Enabling firebug happens on the browser side, so it depends what you are using to launch it. You may be able to use the `Selenium::WebDriver::Firefox::Profile#enable_firebug` method.

**Comment by andrewhavens** [comment]
i am starting the selenium server manually:

```
java -jar selenium-server-standalone-2.0b3.jar
```

**Comment by jfirebaugh** [comment]
Document how #enable_profile can be used with a remote browser.

Closed by 40614cfde45d2ca8ad58dab1ebde05eee8636df7
